app/actions/local/file.ts

```diff
@@ -33,3 +37,34 @@ export const updateLocalFilePath = async (serverUrl: string, fileId: string, loc
         return {error};
     }
 };
+
+export const getAllFilesInCachesDirectory = async () => {
```

Comment:
Why haven't we moved this function away from this file yet? Is the only thing preventing me from continuing my review.

This function does not belong here at all!!

1. Is more an utility than an action
2. Does access the app database and not a server database
Can we plase move it to the files utility or somewhere else? It just not fit here